src/os_posix/os_fs.c

```diff
@@ -858,6 +858,7 @@ __posix_open_file(WT_FILE_SYSTEM *file_system, WT_SESSION *wt_session, const cha
     WT_ERR(__wt_strdup(session, name, &file_handle->name));
 
     pfh->mmap_prot = LF_ISSET(WT_FS_OPEN_READONLY) ? PROT_READ : PROT_READ | PROT_WRITE;
+    pfh->mmap_flags = LF_ISSET(WT_FS_OPEN_READONLY) ? MAP_PRIVATE : MAP_SHARED;
```

Comment:
I'm having difficulty understanding this change. Could you please provide some explanation?

Reply:
Yeah, no problem - for the chunk cache, we want `MAP_SHARED` so that our writes become visible in the file. It was hard-coded to `MAP_PRIVATE` by `__wt_posix_map`, so we needed to make this a little more flexible. This wasn't a correctness issue yet since we aren't expecting the file content to persist, but better to fix it early.